


raise Exception('This exception was raised')

print("Code reached the end") # doesnt reach here
